cmd/install/install.go

```diff
@@ -265,6 +268,17 @@ func NewCommand() *cobra.Command {
 	cmd.PersistentFlags().StringVar(&opts.RegistryOverrides, "registry-overrides", "", "registry-overrides contains the source registry string as a key and the destination registry string as value. Images before being applied are scanned for the source registry string and if found the string is replaced with the destination registry string. Format is: sr1=dr1,sr2=dr2")
 	cmd.PersistentFlags().StringSliceVar(&opts.PlatformsToInstall, "limit-crd-install", opts.PlatformsToInstall, "Used to limit the CRDs that are installed to a per platform basis (example: --limit-crd-install=AWS,Azure). If this flag is not specified, all CRDs for all platforms will be installed. Valid, case-insensitive values are: AWS, Azure, IBMCloud, KubeVirt, Agent, OpenStack.")
 
+	cmd.PersistentPreRunE = func(cmd *cobra.Command, args []string) error {
+		if len(opts.ImagePullPolicy) > 0 {
```

Comment:
Please move this to the [Validate](https://github.com/openshift/hypershift/pull/6875/files#diff-b105e0fd290117fe4056a1e2270006e9ea7a6025f5b57936dff42d50660e0b9cR134) function